src/main/java/br/com/alura/comex/RelatorioSintetico.java

```diff
@@ -0,0 +1,51 @@
+package br.com.alura.comex;
+
+import java.math.BigDecimal;
+import java.util.Comparator;
+import java.util.List;
+
+public class RelatorioSintetico {
+
+    int totalDeProdutosVendidos = 0;
+    int totalDePedidosRealizados = 0;
+    BigDecimal montanteDeVendas = BigDecimal.ZERO;
+    Pedido pedidoMaisBarato;
+    Pedido pedidoMaisCaro;
+
+    CategoriasProcessadas categoriasProcessadas = new CategoriasProcessadas();
+    int totalDeCategorias = 0;
+
+    public RelatorioSintetico(List<Pedido> listaPedidos) {
+        this.totalDeProdutosVendidos = getTotalDeProdutosVendidos(listaPedidos);
+        this.totalDePedidosRealizados = getTotalDePedidosRealizados(listaPedidos);
+        this.montanteDeVendas = getMontanteDeVendas(listaPedidos);
+        this.pedidoMaisBarato = getPedidoMaisBarato(listaPedidos);
+        this.pedidoMaisCaro = getPedidoMaisCaro(listaPedidos);
+        this.totalDeCategorias = getTotalDeCategorias(listaPedidos);
+    }
+
+    public int getTotalDeProdutosVendidos(List<Pedido> listaPedidos) {
+        return totalDeProdutosVendidos = listaPedidos.stream().mapToInt(pedido -> pedido.getQuantidade()).sum();
```

Comment:
Daria pra usar Method Reference!

```suggestion
        return totalDeProdutosVendidos = listaPedidos.stream().mapToInt(Pedido::getQuantidade).sum();
```